Composer/packages/client/src/store/action/lu.ts

```diff
@@ -83,8 +83,8 @@ export const publishLuis: ActionCreator = async ({ dispatch, getState }, authori
   } catch (err) {
     dispatch({
       type: ActionTypes.PUBLISH_LU_FAILED,
-      payload: { title: Text.LUISDEPLOYFAILURE, message: err.response.data.message },
+      payload: { title: Text.LUISDEPLOYFAILURE, message: err.message || err.response.data.message },
     });
-    throw new Error(err.response.data.message);
+    throw err;
```

Comment:
This is an anti-pattern. Our ActionCreators should not not return anything or throw. That's what the dispatcher is for.

Can we keep line 86 and delete 88 and make sure no application code is relying on this behavior?

Reply:
Hi, @a-b-r-o-w-n I have do some refactor about these code and set botstatus to update the ui and logic in test flow.